File: sort-algorithms/selection.js
function selectionSort(ar) {
  for (let i = 0; i <= ar.length - 1; i++) {
    let min = i;
    for (let j = i + 1; j <= ar.length - 1; j++) {
      if (ar[j] < ar[min]) {
        min = j;
      }
    }
    if (min != i) {
      let temp = ar[i];
      ar[i] = ar[min];
      ar[min] = temp;
    }
    console.log(ar);
  }
  // console.log(ar);
}

selectionSort([3, 2, 5, 0, 7]);

File: sort-algorithms/selection.py
def selectionSort(lst):
    for i in range(len(lst) - 1):
        min = i
        for j in range(i+1, len(lst), 1):
            if lst[j] < lst[min]:
                min = j
        if min != i:
            lst[i], lst[min] = lst[min], lst[i]
    print(lst)
    return lst

selectionSort([5, 4, 3, 7, 2, 1])